src/ts/app/editor/src/subscribe.ts

```diff
@@ -0,0 +1,17 @@
+import {Subtree} from "./subtree"
+import {render} from "./render"
+import {CreateSubscriptionFunction, ShadowRootContext} from "app/framework"
```

Comment:
I really like how everything is imported either from the local project, or from root, without relative imports.

Reply:
Yup and I think we can be very consistent about enforcing this. It certainly helps me stay sane when reading code.